### ISSUE TITLE
No module named 'neoteroi.markdown.tables'

### ISSUE THREAD

**Issue by StarfallProjects** [Neoteroi/mkdocs-plugins#4]
Following the README, I've done `pip install neoteroi-mkdocs`, and then enabled both the mkdocsoad plugin, and the spantables extension. However, I'm getting:

```
ERROR    -  Config value: 'markdown_extensions'. Error: No module named 'neoteroi.markdown.tables'
Aborted with 1 Configuration Errors!
```

Is this required for generating the open API reference? Or is it entirely separate?

**Comment by RobertoPrevato** [comment]
Hi @StarfallProjects,
Thank You for reporting this issue! It's a mistake I did in the `setup.py` file, I am fixing this immediately. Sorry for that.

**Comment by RobertoPrevato** [comment]
This is now fixed, together with #7. Please try again with version 0.0.4 for the SpanTable extension.
